### PR TITLE
[ADD] Added ported partner_contact_configuration module.

### PR DESCRIPTION
**Pull request by gbaumgartner-callino**
No changes were needed

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please preserve commit history following technical method explained in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0

File: partner_contact_configuration/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+{
+    'name': 'Partner Contact Configuration',
+    'summary': """
+        Adds menu configuration access through the 'contacts' module main menu
+        """,
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
+    'website': 'https://acsone.eu',
```
Comment (nikul-serpentcs) [inline partner_contact_configuration/__manifest__.py:13]: @gbaumgartner-callino URL must be `https://github.com/OCA/partner-contact`

File: partner_contact_configuration/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    'depends': ['base',
+                'contacts'
+                ],
+    'data': ['views/partner_contact.xml'
+             ],
+    'demo': [
```
Comment (nikul-serpentcs) [inline partner_contact_configuration/__manifest__.py:19]: @gbaumgartner-callino IMO If no need then remove it

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Improve Code

**Comment by rafaelbn** [comment]
Hello @gbaumgartner-callino , Please preserve commit history following technical method explained in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0

Are you going to fix this?

**Comment by NL66278** [comment]
Change requests have not been taken up.

**Pull request closed without merging**